### ISSUE TITLE
[Issue]: Controller not working after settings menu

### ISSUE THREAD

**Issue by patrickdundas** [emukidid/swiss-gc#925]
### Describe the bug

I am running swiss via sd2sp2 with picoboot. Upon launching into swiss for the first time, my official DOL-003 controller works as expected in the initial settings menu popup. Upon closing the initial settings menu via the discard / save buttons, the controller does not respond in the main swiss menu at all. Unplugging the controller or restarting the gamecube does not fix the issue. Starting the gamecube holding down on the dpad shows the debug menu as expected. The controller works in the vanilla menu. Swiss clock continues ticking and temperature is shown as expected, so the UI is not frozen

Tested in Swiss r1742 and r1712

### Steps to reproduce

- Plug in 128gb Sandisk sd card with `ipl.dol` file (copied and renamed official `Swiss/DOL/Swiss_*.dol` file) via sd2sp2
- Plug in DOL-003 controller
- Turn on DOL-001 gamecube with picoboot while holding down on dpad
- Confirm debug menu is shown, so controller dpad input was recognized
- Let swiss boot up and show inital settings popup
- Navigate and save / discard initial settings
- Controller freezes, no dpad or button input is recognized

### Expected behavior

Controller is usable in swiss after being usable in settings menu

### Relevant log output

```shell
gekkoboot r9.2 (with M.2 Loader support)
Trying USB Gecko in Slot B
Not present
Trying USB Gecko in Slot A
Not present
No M.2 Loader hardware detected
Trying sd2
Mounted  as sd2
Reading /z.dol
Failed to open file: Could not find the file
Reading /ipl.dol
Memory available: 227409928
DOL size is 6878808
Reading /ipl.cli
CLI file not found
Unmounting sd2
(release d-pad down to continue)
```

**Comment by kollette27** [comment]
Have you figured this out yet? Mine just started doing this yesterday and I'm literally lost.
My setup is slightly different than yours tho
GameCube DOL-101
Modded w/GC Loader in place of a broken disc drive
No second serial port on this unit, so no sd2sp2
Booting Swiss through 128gb sandisk SD card. 
Flawless performance since February of 2024 and game saving to memory card
-No updates since then and SD card contents before attempting to update both swiss & GC Loader to fix the controller issue
was SD card loaded with swiss_r1614.dol, NKit_v1.4.20240226 & GC Loader v1.0.0
Then Monday 1/13/2025, I went to play & turned it on, only to be met with neither of my controllers working to get me out of swiss menu.
I unplugged controllers and tried them in my wii, they both worked perfectly.
Then ejected the SD card from the loader and plugged controllers back into the GC, they both worked perfectly within the GC bios for erasing memory cards and all.
Then reloaded the Swiss SD card into the GC loader. When swiss loaded, I still had no controller function. Narrowing it down; it's GOT to be a fw/sw issue with the GC mod, GC Loader or Swiss.
I attempted to add update files of both to the SD, but no controller function still, so I couldn't update GC Loader OR Swiss.

My next attempt is to wipe the SD card and reload it with all the updates, as if I'm just now installing the GC Loader HW, but with the updated SW. I really hope I don't have to do this with every update of either Swiss or GC Loader and that my controller function doesn't just quit when the SW needs an update.

**Comment by patrickdundas** [comment]
Yeah this one was a little weird. It eventually just stopped happening after power cycling the GameCube a few times, booting with controllers unplugged or plugged in but not touching the controls for a couple minutes. I sort of just power cycled things in a few different configurations and it sorted itself out - I have no idea what fixed it. I hope yours does the same.   
  
  
  
  
  
  
  

  
  
>

**Comment by patrickdundas** [comment]
No I don’t   
  
  
  
  
  
  
  

  
  
>
